templates/map/index.html.twig

```diff
@@ -24,13 +24,37 @@ mapboxgl.accessToken = 'pk.eyJ1Ijoic2FuY2hlejA4MjMwIiwiYSI6ImNrazJxNGt6eDEzZW4yb
 var map = new mapboxgl.Map({
 container: 'map',
 style: 'mapbox://styles/sanchez08230/ckk3swi694qlc17p855bf8k6h',
-center: [12.550343, 55.665957],
-zoom: 8
+center: [4.031696, 49.258329],
+zoom: 11
 });
- 
-var marker = new mapboxgl.Marker()
-.setLngLat([12.550343, 55.665957])
-.addTo(map);
+{% if app.user.address %}
+fetch("https://api.mapbox.com/geocoding/v5/mapbox.places/{{ app.user.address }}.json?limit=1&access_token=pk.eyJ1Ijoic2FuY2hlejA4MjMwIiwiYSI6ImNrazJxNGt6eDEzZW4yb3J1MmlnbWoyM2IifQ.sL93OELuUpqzZSHwQGes0A")
+        .then(response => response.json())
+        .then(json => {
+            console.log(json, json.features[0].center);
+            const center = json.features[0].center;
+            new mapboxgl.Marker({
+                    color: '#c07a7f'
+                })
+            .setPopup(new mapboxgl.Popup().setHTML("<h1>Mon adresse</h1><p>{{app.user.address}}</p>")) // add popup
+            .setLngLat(center)
+            .addTo(map);
+        });
+{% endif %}
+{% if events %}
+    {% for event in events %}
+        fetch("https://api.mapbox.com/geocoding/v5/mapbox.places/{{ event.place }}.json?limit=1&access_token=pk.eyJ1Ijoic2FuY2hlejA4MjMwIiwiYSI6ImNrazJxNGt6eDEzZW4yb3J1MmlnbWoyM2IifQ.sL93OELuUpqzZSHwQGes0A")
+            .then(response => response.json())
+            .then(json => {
+                console.log(json, json.features[0].center);
+                const center = json.features[0].center;
+                new mapboxgl.Marker()
+                .setPopup(new mapboxgl.Popup().setHTML("<h1>{{event.name}}</h1><p>{{event.place}}</p><p>Début: {{event.datetimeStart|date('d/m/Y H:i')}}</p><p>Fin: {{event.datetimeEnd|date('d/m/Y H:i')}}</p>")) // add popup
+                .setLngLat(center)
+                .addTo(map);
+            })
+    {% endfor %}
```

Comment:
try this:

    const places = [
    {% for event in events %}
        {{ event.place }},
    {% endfor %}
    ]
    places.forEach(place => {
        fetch(/* place */).then(...)
    });